libcxx/test/libcxx/language.support/support.dynamic/libcpp_deallocate.sh.cpp

```diff
@@ -21,6 +21,9 @@
 // GCC doesn't support the aligned-allocation flags.
 // XFAIL: gcc
 
+// TODO(mordante) fix this test after updating clang in Docker
+// UNSUPPORTED: clang-15, clang-16, clang-17, clang-18
```

Comment:
I hope to start testing with clang-19 soon, that might be before this patch lands.
```suggestion
// UNSUPPORTED: clang-15, clang-16, clang-17, clang-18, clang-19
```